docs/source/recipes.rst

```diff
@@ -262,7 +262,7 @@ Finally, instead of modifying the operation implementation, you could use a cust
 
 Storing the above template in a file called ``templates/script.sh`` within your project root directory will prepend *every* operation command with ``mpiexec`` and so on.
 
-How to enforce the execution of a specific operation for debugging
+Forcing the execution of a specific operation for debugging
```

Comment:
```suggestion
Enforcing the execution of a specific operation for debugging
```

Reply:
"Forcing" is probably the expected term here. Resolving.